src/common/utils/Types.h

```diff
@@ -18,7 +18,7 @@ enum class NebulaKeyType : uint32_t {
   kSystem = 0x00000004,
   kOperation = 0x00000005,
   kKeyValue = 0x00000006,
-  // kVertex = 0x00000007,
+  kVertex = 0x00000007,
```

Comment:
Will storing tag/vertex in separate places decrease performance of getting properties of vertex?

Reply:
There will be a decline, but it is negligible for the following reasons:
1. Only when the tag to be read does not exist, will the vertex be judged based on the vertexKey. The probability of this situation is very low.
2. Even if the vertexKey is read, it is only a `get()` request, and the overhead is very small.
3. Someone previously proposed a solution to put `vertexKey` and `tagKey` together, and use the method of `prefix` for query. But the performance of rocksdb's `seek()` and `iterate()` is very poor, which may be dozens of times that of `get()`. This approach will bring performance degradation instead. Of course this is just a theoretical analysis, I will do a simple experiment if necessary.

Reply:
ok